run/config.go

```diff
@@ -495,7 +496,7 @@ func loadConfig(cfg *Config) error {
 		}
 
 		if !tempWalletExists {
-			// Perform the initial wallet creation wizard.
+			// Create the simulation wallet using provided config.
```

Comment:
it's not simulation wallet per se, it is a wallet ran on simnet, so simnet wallet would be more explanatory